suite-native/module-send/src/sendFormThunks.ts

```diff
@@ -87,6 +89,21 @@ export const cleanupSendFormThunk = createThunk(
     },
 );
 
+export const cleanupBitcoinSendFormsThunk = createThunk(
```

Comment:
I would prefer a better name. This suggests it cleans up either:
1. only BTC send forms
2. send forms of all BTC-like coins

None of the above applies. I suggest something like cleanupSatoshisSuportingSendFormsThunk etc.